internal/querycoordv2/observers/leader_observer.go

```diff
@@ -184,6 +184,12 @@ func (o *LeaderObserver) findNeedLoadedSegments(leaderView *meta.LeaderView, dis
 			}
 			loadInfo := utils.PackSegmentLoadInfo(resp, nil)
 
+			log.Debug("leader observer append a segment to set:",
```

Comment:
Also the log will be formatted, so the ":" is not needed. Correct "append" to "appends"